rust/blockstore/src/arrow/ordered_blockfile_writer.rs

```diff
@@ -133,6 +148,20 @@ impl ArrowOrderedBlockfileWriter {
 
         Self::complete_current_delta::<K, V>(&mut inner);
 
+        let num_block_cache_hits = self
+            .num_block_cache_hits
+            .load(std::sync::atomic::Ordering::Relaxed);
+        let num_block_cache_misses = self
+            .num_block_cache_misses
+            .load(std::sync::atomic::Ordering::Relaxed);
+        let block_cache_hit_rate = 100.0 * num_block_cache_hits as f64
+            / (num_block_cache_hits + num_block_cache_misses) as f64;
+        tracing::debug!(
+            block_cache_hit_rate = block_cache_hit_rate,
+            "Block cache hit rate: {:.2}%",
```

Comment:
this was useful when implementing--in practice this should always be 100% for the ordered blockfile writer since it's only used for FTS, but I bet this would be useful to have for the unordered blockfile writer as well
can remove to cut scope and introduce in separate PR if people feel that's cleaner